src/components/HomePage/BlogList.js

```diff
@@ -7,8 +7,9 @@ import { withStyles } from '@material-ui/core/styles';
 import BlogListItem from './BlogListItem';
 
 const styles = {
-	listItem: {
-		overflowWrap: 'break-word'
+	root: {
```

Comment:
This should still be called `listItem`, as it’s applied to the root of a list item.